test/functional/explain.test.js

```diff
@@ -0,0 +1,429 @@
+'use strict';
+const { setupDatabase, withClient } = require('./shared');
+const chai = require('chai');
+const expect = chai.expect;
+chai.use(require('chai-subset'));
+
+describe('Explain', function () {
+  before(function () {
+    return setupDatabase(this.configuration);
+  });
+
+  it('shouldHonorBooleanExplainWithDeleteOne', {
```

Comment:
Nit: test names for new tests should be written in normal text with spaces rather than camel case.